e2e/utils_test.go

```diff
@@ -211,7 +211,6 @@ func setupCluster(configFiles ...string) ([]*gdProcess, error) {
 func teardownCluster(gds []*gdProcess) error {
 	for _, gd := range gds {
 		gd.Stop()
-		gd.EraseWorkdir()
```

Comment:
`setupCluster()` and `teardownCluster()` are not used by all tests. You may want to fix this (remove EraseWorkdir) at other places too for completeness.

```sh
[ppai@gd2-1 glusterd2]$ grep -nR "EraseWorkdir" . | grep -v utils
grep: warning: ./vendor/github.com/coreos/etcd/cmd/etcd: recursive directory loop
./e2e/peer_ops_test.go:16:	defer g1.EraseWorkdir()
./e2e/peer_ops_test.go:22:	defer g2.EraseWorkdir()
./e2e/peer_ops_test.go:28:	defer g3.EraseWorkdir()
./e2e/restapi_auth_test.go:18:	defer g1.EraseWorkdir()
```